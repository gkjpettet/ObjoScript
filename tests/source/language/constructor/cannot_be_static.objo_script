class Foo {
  static constructor() {} // expect error
}