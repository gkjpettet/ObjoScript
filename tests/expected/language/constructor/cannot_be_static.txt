Expected lowercase identifier but got constructor instead.